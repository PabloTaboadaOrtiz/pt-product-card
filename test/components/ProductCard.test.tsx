import * as React from 'react';
import renderer from 'react-test-renderer';
import { ProductCard } from '../../src/components';
import { PRODUCT_2 } from '../data/products';

describe('ProductCard', () => {
  test('debe de mostar el componente', () => {
    const wrapper = renderer.create(
      <ProductCard product={PRODUCT_2}>
        {() => <h1>Product card</h1>}
      </ProductCard>
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
