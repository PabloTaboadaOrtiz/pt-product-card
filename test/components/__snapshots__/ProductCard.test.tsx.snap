// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCard debe de mostar el componente 1`] = `
<div
  className="productCard undefined"
>
  <h1>
    Product card
  </h1>
</div>
`;
